src/main/kotlin/org/anefdev/mixbuddy/util/PlaylistParser.kt

```diff
@@ -0,0 +1,168 @@
+package org.anefdev.mixbuddy.util
+
+import org.anefdev.mixbuddy.model.MusicPlaylist
+import org.anefdev.mixbuddy.model.MusicTrack
+import org.springframework.stereotype.Component
+import se.michaelthelin.spotify.model_objects.miscellaneous.AudioAnalysis
+import se.michaelthelin.spotify.model_objects.specification.PlaylistSimplified
+import se.michaelthelin.spotify.model_objects.specification.PlaylistTrack
+import se.michaelthelin.spotify.model_objects.specification.Track
+import kotlin.math.abs
+
+@Component
+class PlaylistParser {
+
+    fun getTrackListNoMetaData(playlist: List<PlaylistTrack>): List<MusicTrack> {
+        return playlist.stream().map { playlistTrack: PlaylistTrack ->
```

Comment:
doesnt seem that having named lambda parameter is necessary here, especially with an explicit type